lib/ex_webrtc/rtp_receiver/simulcast_demuxer.ex

```diff
@@ -0,0 +1,94 @@
+defmodule ExWebRTC.RTPReceiver.SimulcastDemuxer do
+  @moduledoc false
+
+  alias ExSDP.Attribute.Extmap
+
+  alias ExRTP.Packet
+  alias ExRTP.Packet.Extension
+  alias ExRTP.Packet.Extension.SourceDescription
+
+  @rid_uri "urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id"
+  @rrid_uri "urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id"
+
+  @type t() :: %__MODULE__{
+          rid_ext_id: non_neg_integer() | nil,
+          rrid_ext_id: non_neg_integer() | nil,
+          ssrc_to_rid: %{(ssrc :: non_neg_integer()) => rid :: String.t()}
+        }
```

Comment:
Our demuxer does not check whether received rid was specified in SDP offer so in theory, if in SDP we have `h,m,l` and we receive a packet with e.g. `a` we will handle such packet. Am I correct?

Reply:
Yeah, I decided it isn't much of an issue.